cli.js

```diff
@@ -153,6 +157,27 @@ if (files.length === 0) {
 	];
 }
 
+var timeout;
+var timer;
+
+function restartTimer() {
+	clearTimeout(timer);
+
+	timer = setTimeout(onTimeout, timeout);
```

Comment:
Use https://github.com/sindresorhus/ava/blob/8d47119458e83d3899683ad3ea3a4c1c01b7dd49/lib/globals.js#L9 to prevent it being mocked with when people use Sinon/etc. Same with `clearTimeout`.